lms/djangoapps/grades/events.py

```diff
@@ -209,6 +257,45 @@ def course_grade_now_failed(user, course_id):
             }
         )
 
+    # produce to event bus
```

Comment:
Can this be moved to the utility function, so the condition for checking course type isn't duplicated?